.github/dependabot.yml

```diff
@@ -3,7 +3,6 @@ updates:
   - package-ecosystem: 'npm'
     directories:
       - '/'
-      - '/ws-nextjs-app'
```

Comment:
If we do this, would it mean you won't get updates for NextJS & undici, which are currently only declared in   `/ws-nextjs-app`?

I have also been looking into this, and trying to prevent the duplicate notifications - see #11865 - is this a better option?

Reply:
Do you want to try this first... and keep an eye on the undici & Next packages - which seem to be fairly actively developed at the moment, as we see regular updates? And then we could have a think about the shared dependency PR above...?

Reply:
Worth trying this, as I feel like `directories` is more for mono-repos with `package.json` files in separate folders. The Next.js one is nested within `/`, so it may be getting confused: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directories

Reply:
Let's do it 🚀